src/Chapter4.hs

```diff
@@ -610,6 +642,14 @@ Implement the 'Monad' instance for our lists.
 🕯 HINT: You probably will need to implement a helper function (or
   maybe a few) to flatten lists of lists to a single list.
 -}
+join :: List (List a) -> List a
+join Empty       = Empty
+join (Cons x xs) = x `append` (join xs)
+
+instance Monad List where
+    (>>=) :: List a -> (a -> List b) -> List b
+    (>>=) Empty _ = Empty
```

Comment:
This case is redundant, because both `join` and `fmap` return `Empty` on `Empty` list :slightly_smiling_face: